File: cs/Iris/ExpressionElement/DefaultExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ExpressionElement
{
    public class DefaultExpression : SwitchCaseExpression, IBlockExpression
    {
        public Expression body;

        public DefaultExpression()
        {

        }

        public List<string> declaredVariables()
        {
            return null;
        }
    }
}

File: java/Iris/src/com/rokuan/iris/expression/DefaultExpression.java
package com.rokuan.iris.expression;

import java.util.ArrayList;

public class DefaultExpression extends SwitchCaseExpression implements IBlockExpression
{
	public Expression body;

	public DefaultExpression()
	{

	}

	public ArrayList<String> declaredVariables()
	{
		return null;
	}

	/*public override string ToString(string alinea)
        {
            StringBuilder str = new StringBuilder();

            str.Append(alinea);
            str.Append("default:\n");

            if (body != null)
            {
                str.Append(body.ToString(alinea + '\t'));
                str.Append('\n');
            }

            str.Append(alinea + '\t');
            str.Append("break;");

            return str.ToString();
        }*/
}
